examples/play_buffer.py

```diff
@@ -1,7 +1,32 @@
 #!/usr/bin/env python3
 
+# Author:   A.Invernizzi (@albestro on GitHub)
```

Comment:
For the license, there are currently 2 different types of licenses for examples (GPLv2+ or MIT), but the best option for examples may indeed be the more permissive MIT license (as in cocoavlc), so you can use cocoavlc as reference if it is OK for you.